### PR TITLE
docs: extend filestash example with more regions

### PR DESCRIPTION
**Pull request by derpsteb**
### Context
Extend docs slightly. 

### Proposed change(s)

- Add all currently defined regions to filestash example.
- Add more explanation what to do with the example.

### Checklist

- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)
- [x] Link to Milestone

### CODE REVIEW

File: docs/docs/getting-started/examples/filestash-s3proxy.md [hunk 1]
```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
 
 Afterward, you can use a port forward to access the Filestash pod:
 `kubectl port-forward pod/$(kubectl get pod --selector='app=filestash' -o=jsonpath='{.items[*].metadata.name}') 8334:8334`
+
+4. After browsing to `localhost:8443`, Filestash will ask you to set an administrator password.
+After setting it you can directly leave the admin area by clicking the blue cloud symbol in the top left corner.
```
Comment (thomasten) [inline docs/docs/getting-started/examples/filestash-s3proxy.md:97]:
```suggestion
After setting it, you can directly leave the admin area by clicking the blue cloud symbol in the top left corner.
```
ms style guide wants comma after introductory phrases

File: docs/docs/getting-started/examples/filestash-s3proxy.md [hunk 2]
```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
...
+Subsequently, you can select S3 as storage backend and enter your credentials.
+This will bring you to an overview of your buckets.
+If you want to deploy Filestash in production, please take a look at it's [documentation](https://www.filestash.app/docs/).
```
Comment (thomasten) [inline docs/docs/getting-started/examples/filestash-s3proxy.md:100]:
```suggestion
If you want to deploy Filestash in production, take a look at its [documentation](https://www.filestash.app/docs/).
```
ms style guide "Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation."

File: docs/docs/getting-started/examples/filestash-s3proxy.md [hunk 3]
```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
...
+
+5. To see the logs of s3proxy intercepting requests made to S3, run: `kubectl logs -f pod/$(kubectl get pod --selector='app=s3proxy' -o=jsonpath='{.items[*].metadata.name}')`
+This will stream the logs of s3proxy to the terminal you are running the command in.
```
Comment (thomasten) [inline docs/docs/getting-started/examples/filestash-s3proxy.md:103]:
```suggestion
```
this should be commonly known by the target audience

File: docs/docs/getting-started/examples/filestash-s3proxy.md [hunk 4]
```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
...
+Look out for log messages labeled "intercepting".
+There is one such log message for each message that's encrypted, decrypted or blocked.
```
Comment (thomasten) [inline docs/docs/getting-started/examples/filestash-s3proxy.md:105]:
```suggestion
There is one such log message for each message that's encrypted, decrypted, or blocked.
```
ms style guide wants oxford comma

File: docs/docs/getting-started/examples/filestash-s3proxy.md [hunk 5]
```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
...
+
+6. Once you have uploaded a file with Filestash you should be able to view the file in Filestash.
+However, if you go to the AWS S3 [Web UI](https://s3.console.aws.amazon.com/s3/home) and download the file you just uploaded in Filestash, you won't be able to read it.
+Another way to spot encrypted files without downloading them is to click on a file, scroll to the Metadata section and look for the header named `x-amz-meta-constellation-encryption`.
```
Comment (thomasten) [inline docs/docs/getting-started/examples/filestash-s3proxy.md:109]:
```suggestion
6. Once you have uploaded a file with Filestash, you should be able to view the file in Filestash.
However, if you go to the AWS S3 [Web UI](https://s3.console.aws.amazon.com/s3/home) and download the file you just uploaded in Filestash, you won't be able to read it.
Another way to spot encrypted files without downloading them is to click on a file, scroll to the Metadata section, and look for the header named `x-amz-meta-constellation-encryption`.
```

**Review by thomasten — APPROVED** [review]
Thanks, that's what I had in mind.
Remember to apply suggestions to both files.

**Comment by derpsteb** [comment]
Ignoring CI failures as they are not related to this PR.

**Pull request merged**